Makefile

```diff
@@ -288,12 +292,12 @@ clean:
 	-rm -f $(PROGRAM)
 
 install: $(PROGRAM)
-	cp $(PROGRAM) /usr/local/bin
```

Comment:
Fix parallel installation by processing commands sequentially.
Use `mkdir -p` for creation of all directories.
Use `cp -a` to preserve build timestamps (probably better to use `install -p`.
Install hspdr_icon.png to the proper system wide place.